docs/dao/index.md

```diff
@@ -0,0 +1,43 @@
+---
+id: dao
+sidebar_label: JAM Implementers DAO
+sidebar_position: 5
+slug: /dao/
+---
+
+# JAM Implementers DAO 
+
+## Background 
+
+JAM Implementers DAO is a community of [JAM Implementers](https://jamcha.in/clients) dedicated to implementing JAM protocols, envisioned to underpin Polkadot’s technical evolution and the future of trustless supercomputing.  
+
+## Aims of the DAO
+
+By implementing JAM and engaging with the Polkadot broader Web3 + trustless supercomputing communities, we aim to advance the Polkadot/JAM  ecosystem and drive real-world adoption of Polkadot+JAM protocols.
+
+## Members
+
+This section functions as the single source of truth of JAM Implementer DAO Membership, supported by on-chain extrinsics.
+
+* [JAM DUNA](https://polkadot.subscan.io/extrinsic/25277967-2)	121Rs6fKm8nguHnvPfG1Cq3ctFuNAVZGRmghwkJwHpKxKjbx 
+* [Gossamer](https://polkadot.subscan.io/extrinsic/25278352-2) 143FppLAgb1KrvTRfkWKnozBeoZsZrhRr41gSBMyUDkUNT8z	
+* [Jamixir](https://polkadot.subscan.io/tx/0x2a66d68acab375502c60a4d36f168e476d7785189fcd916522a0a2666c746588)  15mXN5E3gn14t3etvcGHGfR4zL9WNoCBGCu9rJJiERqjePY8	
+* [JavaJAM](https://polkadot.subscan.io/extrinsic/25317339-2)	14zb7FpKGRCoW8SPezDHsMzqhGAuEn6Wjprf4j2Gn85dGVEh	
+* [JamZig ⚡️](https://polkadot.subscan.io/extrinsic/25319122-2)	14T1UcnoaMND24ubV86FCT4YxJ8q5CKVG7winwZ7umAit8g2	
+* [Jampy](https://polkadot.subscan.io/extrinsic/0x240cbc421e93ba875bb40edf5998a7f85f6da5bfacf4efa2d401a8cd1fe00005)	12iqwZGB2sguEhjFi2ZRuWWixU8mHJnSiP1pwDefqGsBy4rV	
+* [Vinwolf](https://polkadot.subscan.io/extrinsic/25317663-2) 1urZ9pp1D6aL6SRwepP9zhU2kzgxJ3dtRodSLe4paJCpLrk	
+* [TSJam](https://polkadot.subscan.io/extrinsic/25317150-2)	155tk9HmeJGsNZtA5LFasSCGZCdpAb2P2Gs6ej9JeP38sAww	
+
+Additional membership and participation is strongly encouraged by JAM Implementers, subject to _Membership Requirements_ below.  
+
+Extending support to JAM Service builders, tool builders and general JAM evangelist is out of scope at present, but may be pursued in the future.
+
+## Membership Requirements
+
+1. Your team must be registered a JAM Implementer on [jamcha.in/clients](https://jamcha.in/clients).
+2. You have read and agree to abide by [JAM Implementers Code of Conduct](/dao/code_of_conduct).
+3. You submit an on-chain extrinsic (to Polkadot Relay Chain or Asset Hub) with a system.remark `join(JAM Implementers DAO, [your team name])` and add your team to above via a PR.
+
+If there are multiple remarks from the same team or additions/removals based on issues related to Code of Conduct, the admins of this repo may decide these issues unilaterally if clear resolutions reasonably exist or request that existing members decide it based on social proof (participation in online or real-world meetups, GP Chat room, W3F Github repos, etc.).  
```

Comment:
Sounds good. It could also require a set on-chain identity but probably fine to start easy.